src/App.tsx

```diff
@@ -1,25 +1,20 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
-import "./styles/tailwind.output.css";
+import React from 'react';
+import './App.scss';
+import LoginPage from './pages/login/LoginPage';
+import SignupPage from './pages/signup/SignupPage';
+import './styles/tailwind.output.css';
```

Comment:
Should group imports from node_modules together and place them in the top, imports from our source code in the bottom

Reply:
Fixed!